src/js/edu-benefits/5490/config/form.js

```diff
@@ -459,11 +446,7 @@ const formConfig = {
                     properties: {
                       name: highSchool.properties.name,
                       city: highSchool.properties.city,
-                      state: highSchool.properties.state,
-                      dateRange: highSchool.properties.dateRange,
```

Comment:
I'm not sure about this one--see comment in the issue about which options require dateRange vs date vs neither